.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "typescript.tsdk": "node_modules\\typescript\\lib"
```

Comment:
Not sure if this change should be committed

Reply:
It wasn't gitignored, so we should either do that or leave this in. Otherwise, I'd need to add this in locally everytime. I think this setting should be shared though, we should all use the workspace version of TypeScript.

Reply:
Alright, feel free to merge!